src/app/filecoin-plus/allocators/utils/formatDatacap.ts

```diff
@@ -0,0 +1,8 @@
+import prettyBytes from 'pretty-bytes'
+
+export function formatDatacap(bytes?: string) {
+  if (bytes === undefined || bytes === '') return 'N/A'
```

Comment:
Nit: Both `undefined` and `''` are falsy values, so it could simplified as:


```suggestion
  if (!bytes) return 'N/A'
```

But what you have also works fine :)

Reply:
In this case, I think we want to be explicit, because we want to show 0B if the value is 0 (also falsy)

Reply:
Ok, works for me!